apps/remote_control/test/lexical/remote_control/code_intelligence/ast/aliases_test.exs

```diff
@@ -0,0 +1,149 @@
+defmodule Lexical.RemoteControl.CodeIntelligence.Ast.AliasesTest do
+  alias Lexical.Document
+  alias Lexical.Document.Position
+  alias Lexical.RemoteControl.CodeIntelligence.Ast
+
+  import Lexical.Test.CursorSupport
+  import Lexical.Test.CodeSigil
+
+  use ExUnit.Case, async: true
+
+  describe "at_position/2" do
```

Comment:
this test feels light to me given how much code is in aliases. Are you sure all code paths and all the things you've accounted for are tested here?
What about things that weren't included?

Reply:
I have added all the scenarios I can think of. If you have more edge cases, please feel free to let me know.

Reply:
What about aliases that overwrite one another?
Aliases inside of a function definition? 
Aliases inside of an anonymous function and then right after it?
```elixir
alias Something.Else
def my_fn do 
  fn anony = fn -> 
    alias Another.Else
   |
  end
end
```
```elixir
alias Something.Else
def my_fn do 
  fn anony = fn -> 
    alias Another.Else
  end
  |
end
```